code/game/mecha/mecha_control_console.dm

```diff
@@ -122,10 +119,34 @@
 						<b>Active equipment:</b> [M.selected||"None"]<br>"}
 	if(istype(M, /obj/mecha/working/ripley))
 		var/obj/mecha/working/ripley/RM = M
-		answer += "<b>Used cargo space:</b> [RM.cargo.len/RM.cargo_capacity*100]%<br>"
+		answer += "<b>Used cargo space:</b> [length(RM.cargo)/RM.cargo_capacity*100]%<br>"
 
 	return answer
 
+/obj/item/mecha_parts/mecha_tracking/tgui_data(mob/user)
```

Comment:
Dont call this `tgui_data` incase tracking beacons themselves ever get a TGUI for whatever reason. Call it `retrieve_data` or something.